azurelinuxagent/ga/update.py

```diff
@@ -369,6 +370,7 @@ def run(self, debug=False):
             self._ensure_partition_assigned()
             self._ensure_readonly_files()
             self._ensure_cgroups_initialized()
+            self._ensure_policy_initialized()
```

Comment:
I think this should be behind conf flag

Reply:
I think we do need the policy "configurator" initialized even if policy is turned off, so we can have a centralized place to do the conf flag check, instead of doing it in each method. If the flag is set to false, the actual Regorus engine won't be initialized, because the conf flag check is the first thing in PolicyEngine.initialize(). Thoughts?

Reply:
Yup that's where I'm leaning too